Templates/Common-Backend-Scripts/prepareLogs.sh

```diff
@@ -158,6 +159,12 @@ validateOptions() {
         # Compute the outDir parameter
         outDir=$(getLogDir)
 
+        if [ ! -d "$(outDir)" ]; then
+            rc=8
+            ERRMSG=$PGM": [ERROR] The Log directory ($outDir) was not found. Skipping creating tar file. rc="$rc
```

Comment:
Is it "Log" or "Logs"? I believe it is named "logs" when created.
```suggestion
            ERRMSG=$PGM": [ERROR] The '$outDir' logs directory was not found. Skipping creation of tar file. rc="$rc
```